tf2onnx/schema.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+"""
+tf2onnx.schema
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import defaultdict, OrderedDict
+from onnx import defs
+
+def version_mapping(schemas):
+    """Obtain version to schema mapping."""
+    schemas_with_version = defaultdict(dict)
+    for s in schemas:
+        schemas_with_version[s.name][int(s.since_version)] = s
+    for name, s in schemas_with_version.items():
+        schemas_with_version[name] = OrderedDict(
+            sorted(schemas_with_version[name].items(), key=lambda x: x[0])
+        )
+    return schemas_with_version
+
```

Comment:
it might be better to define a schema class, which is a wrapper of onnx schema, may provide some schema level utilities.

Reply:
good point!